app/src/main/java/com/android/wardrobeManager/ClosetActivity.java

```diff
@@ -13,16 +13,17 @@
     boolean shoeScrollVisible = false;
     boolean shortScrollVisible = false;
 
-    String buttonToggleOn = getResources().getString(R.string.closet_toggle_on);
-    String buttonToggleOff = getResources().getString(R.string.closet_toggle_off);
```

Comment:
Instead of closet_toggle_on/off, I think it would be a little more descriptive to be closet_clothing_type_show/hide